sample-input/pin-cell/PC-isotope.py

```diff
@@ -0,0 +1,274 @@
+h1.setSigmaA(np.array(xs['absorption']['average'][:]))
+h1.setSigmaS(np.array(xs['scatter matrix']['average'][:]).flatten('F'))
+
+
+# O-16 in Moderator
+o16m = Isotope(name='O-16 in Moderator')
+o16m.setNumEnergyGroups(8)
+xs = mgxs['material 10003']['O-16']
+o16m.setSigmaT(np.array(xs['transport']['average'][:]))
+o16m.setSigmaA(np.array(xs['absorption']['average'][:]))
+o16m.setSigmaS(np.array(xs['scatter matrix']['average'][:]).flatten('F'))
+
+
+# B-10
+b10 = Isotope(name='B-10')
+b10.setNumEnergyGroups(8)
+xs = mgxs['material 10003']['B-10']
+b10.setSigmaT(np.array(xs['transport']['average'][:]))
+b10.setSigmaA(np.array(xs['absorption']['average'][:]))
+b10.setSigmaS(np.array(xs['scatter matrix']['average'][:]).flatten('F'))
+
+# B-11
+b11 = Isotope(name='B-11')
+b11.setNumEnergyGroups(8)
+xs = mgxs['material 10003']['B-11']
+b11.setSigmaT(np.array(xs['transport']['average'][:]))
+b11.setSigmaA(np.array(xs['absorption']['average'][:]))
+b11.setSigmaS(np.array(xs['scatter matrix']['average'][:]).flatten('F'))
+
+
+
+# Zr-90
+zr90 = Isotope(name='Zr-90')
+zr90.setNumEnergyGroups(8)
+xs = mgxs['material 10005']['Zr-90']
+zr90.setSigmaT(np.array(xs['transport']['average'][:]))
+zr90.setSigmaA(np.array(xs['absorption']['average'][:]))
+zr90.setSigmaS(np.array(xs['scatter matrix']['average'][:]).flatten('F'))
+
+
+# Zr-91
+zr91 = Isotope(name='Zr-91')
+zr91.setNumEnergyGroups(8)
+xs = mgxs['material 10005']['Zr-91']
+zr91.setSigmaT(np.array(xs['transport']['average'][:]))
+zr91.setSigmaA(np.array(xs['absorption']['average'][:]))
+zr91.setSigmaS(np.array(xs['scatter matrix']['average'][:]).flatten('F'))
+
+
+# Zr-92
+zr92 = Isotope(name='Zr-92')
+zr92.setNumEnergyGroups(8)
+xs = mgxs['material 10005']['Zr-92']
+zr92.setSigmaT(np.array(xs['transport']['average'][:]))
+zr92.setSigmaA(np.array(xs['absorption']['average'][:]))
+zr92.setSigmaS(np.array(xs['scatter matrix']['average'][:]).flatten('F'))
+
+
+# Zr-94
+zr94 = Isotope(name='Zr-94')
+zr94.setNumEnergyGroups(8)
+xs = mgxs['material 10005']['Zr-94']
+zr94.setSigmaT(np.array(xs['transport']['average'][:]))
+zr94.setSigmaA(np.array(xs['absorption']['average'][:]))
+zr94.setSigmaS(np.array(xs['scatter matrix']['average'][:]).flatten('F'))
+
+
+# Zr-96
+zr96 = Isotope(name='Zr-96')
+zr96.setNumEnergyGroups(8)
+xs = mgxs['material 10005']['Zr-96']
+zr96.setSigmaT(np.array(xs['transport']['average'][:]))
+zr96.setSigmaA(np.array(xs['absorption']['average'][:]))
+zr96.setSigmaS(np.array(xs['scatter matrix']['average'][:]).flatten('F'))
+
+
+###############################################################################
+###########################   Creating Materials   ############################
+###############################################################################
+
+log.py_printf('NORMAL', 'Creating materials...')
+
+# Fuel
+fuel_matl = IsoMaterial(name='Fuel')
+fuel_matl.addIsotope(u235,3.7503e-4)
+fuel_matl.addIsotope(u238,2.2625e-2)
```

Comment:
It's pretty nitpicky, but our Style Guidelines do call for a space between function parameters ([link](https://mit-crpg.github.io/OpenMOC/devguide/style_guide.html#whitespace))